src/Umbraco.Core/Composing/TypeLoader.cs

```diff
@@ -505,6 +505,38 @@ private void DeleteFile(string path, int timeoutMilliseconds)
 
         #endregion
 
+        #region Get Assembly Attributes
+
+        /// <summary>
+        /// Gets the assembly attributes.
+        /// </summary>
+        /// <typeparam name="T">The attribute type.</typeparam>
+        /// <returns>
+        /// The assembly attributes of the specified type.
+        /// </returns>
+        public IEnumerable<T> GetAssemblyAttributes<T>()
+            where T : Attribute
+        {
+            return AssembliesToScan.SelectMany(a => a.GetCustomAttributes<T>());
+        }
+
+        /// <summary>
+        /// Gets the assembly attributes.
+        /// </summary>
+        /// <param name="attributeTypes">The attribute types.</param>
+        /// <returns>
+        /// The assembly attributes of the specified types.
+        /// </returns>
+        /// <exception cref="ArgumentNullException">attributeTypes</exception>
+        public IEnumerable<Attribute> GetAssemblyAttributes(params Type[] attributeTypes)
```

Comment:
This method currently returns zero attributes if no types are supplied (empty array/no arguments). Should we instead return all assembly attributes in that case? That would align more with how `GetCustomAttributes()` works, but might cause issues if the `attributeTypes` is dynamically generated... Thoughts?

Reply:
I've fixed this by adding a parameterless overload 👍